### DISCUSSION TITLE
[1164250]胶佬模拟器 {仅本体}

### DISCUSSION THREAD

**Discussion by Bokyer** [pjy612/SteamManifestCache#797] [Share]
### Steam 游戏ID

1164250

### 密钥列表

```hash
1164251;d83fb99cf6eb80bbbac0b70711f33b547592a4c48142869a6641ed21ec1c7cbe
```

### 清单附件

[1164251_4673428734292625881.zip](https://github.com/user-attachments/files/16919231/1164251_4673428734292625881.zip)

### 备注

383300900@qq.com

**Comment by ZYsuai** [comment]
e宝送过了没什么必要

**Comment by ZEERDEER** [comment]
* 清单附件没有上传完成

**Comment by pjy612** [comment]
请确保附件上传完成 再提交

**Reply by Bokyer** [reply]
已重新上传
